Allow creating instance without timer or data (#5112)
rathena/rathena: doc/script_commands.txt

---------------------------------------

*instance_list(<instance mode>,<"map name">);

> Set '.@instance_list' asan array for the instances ids with the map name and the mode passed.

Reviewer: ```suggestion
Set '.@instance_list' as an array for the instances ids with the map name and the mode passed.
```

---------------------------------------

*instance_list(<"map name">{,<instance mode>});

> Set '.@instance_list' as an array for the instances ids with the map name and the mode passed.

Reviewer 2: ```suggestion
Creates the array '.@instance_list' with possible instance IDs for the given <map name> and optional <mode>.
```

Return '.@instance_list' array size.

> instance mode IM_NONE, IM_CHAR, IM_PARTY, IM_GUILD, IM_CLAN
> if the instance mode is not provided By default it will return all the instances ids for that map

Reviewer 2: ```suggestion
Instance mode options: IM_NONE, IM_CHAR, IM_PARTY, IM_GUILD, or IM_CLAN
If the instance mode is not provided then it will return all the instance IDs for that map.
```

Examples:

> 	// this example assums that you already have punch of prontera instances.

Reviewer 2: ```suggestion
	// This example assumes that there are several instances on the map of Prontera.
```
